openhantek/src/widgets/dsowidget.h

```diff
@@ -24,10 +24,11 @@ struct ModelSpec;
 class GlScopeWindow;
 class DsoControl;
 
-class ChannelWidgets : protected QWidget {
+class ChannelWidgets : public QWidget {
+  Q_OBJECT
```

Comment:
We are not using the old Signals/Slots syntax or Qt reflection on this class, so strictly speaking Q_OBJECT is not required. But doesn't harm at the moment, so keep it.